### PR TITLE
Add kitspace.yml for RevC1

### PR DESCRIPTION
**Pull request by John-K**
Per discussion in #193 switch to using the "multi project" kitspace yml format.

### CODE REVIEW

**Comment by John-K** [comment]
@whitequark does this look more suitable for future revisions?

File: kitspace.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+multi:
+    Glasgow_RevC1:
```
Comment (whitequark) [inline kitspace.yml:2]:
```suggestion
    Glasgow_revC1:
```

File: kitspace.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+        gerbers: hardware/boards/glasgow/revC1
+        bom: hardware/boards/glasgow/revC1/bom.csv
+        color: blue
```
Comment (whitequark) [inline kitspace.yml:5]: Color of what?

Comment (kasbah) [reply kitspace.yml:5]: It's just the solder mask color of the preview rendering of the board. It has no other effect beyond that.

Comment (whitequark) [reply kitspace.yml:5]: Let's go for green then, like in README.

File: kitspace.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+multi:
+    Glasgow_revC1:
+        gerbers: hardware/boards/glasgow/revC1
+        bom: hardware/boards/glasgow/revC1/bom.csv
+        color: blue
```
Comment (John-K) [inline kitspace.yml:5]:
```suggestion
        color: green
```

File: kitspace.yml (hunk 2 above)
Comment (John-K) [reply kitspace.yml]: done

**Pull request merged**